activesupport/lib/active_support/core_ext/module/anonymous.rb

```diff
@@ -7,7 +7,7 @@ class Module
   #   m = Module.new
   #   m.name # => nil
   #
-  # +anonymous?+ method returns true if module does not have a name:
+  # +anonymous?+ method returns +true+ if module does not have a name, +false+ otherwise:
```

Comment:
:scream:  They are [English words rather than Ruby keywords](http://edgeguides.rubyonrails.org/api_documentation_guidelines.html#regular-font) ..... sorry